File: src/main/kotlin/com/gas/vesubio/Model/Inf_Valores.java
package com.gas.vesubio.Model;

import javax.persistence.*;

@Entity
public class Inf_Valores {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long val_id;
    @Column
    private String var_descripcion;

    public Long getVal_id() {
        return val_id;
    }

    public void setVal_id(Long val_id) {
        this.val_id = val_id;
    }

    public String getVar_descripcion() {
        return var_descripcion;
    }

    public void setVar_descripcion(String var_descripcion) {
        this.var_descripcion = var_descripcion;
    }
}

File: src/main/kotlin/com/gas/vesubio/Model/Inf_Valores.kt
package com.gas.vesubio.Model

import java.io.Serializable
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "Inf_Inf_Valores")
class Inf_Valores: Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    @Column(name = "var_id")
    val id: Long? = 0
    @Column(name = "var_descrip", nullable = false)
    val descripcion:String? = null
}